designs/disruption-controls-drain-timeout.md

```diff
@@ -0,0 +1,78 @@
+# Disruption Controls - (in v1beta1+)
```

Comment:
Thoughts on integrating with https://kubernetes.io/blog/2022/12/16/kubernetes-1-26-non-graceful-node-shutdown-beta/#how-does-it-work

Reply:
This looks promising, it looks like it also handles the volume detachment case too.

Once the timeout is reach, I think karpenter would need to set the taint, then watch for all pods to be forcibly terminated, then clean up the node on the cloud provider's side.

Reply:
Updated and added the non-graceful node shutdown doc as a consideration in the behavior implementation section.